website/docs/r/sql_database.html.markdown

```diff
@@ -37,6 +37,10 @@ The following arguments are supported:
 
 * `instance` - (Required) The name of containing instance.
 
+* `charset` - (Optional) The MySQL charset value.
```

Comment:
Would you mind adding the `charset` and `collation` fields to the example at the top of the page? (Apologies for commenting here - Github doesn't like comments on unchanged lines)